pkg/cmd/start/start.go

```diff
@@ -69,36 +69,36 @@ func NewGather() *cobra.Command {
 func runGather(operator *controller.GatherJob, cfg *controllercmd.ControllerCommandConfig) func(cmd *cobra.Command, args []string) {
 	return func(cmd *cobra.Command, args []string) {
 		if configArg := cmd.Flags().Lookup("config").Value.String(); len(configArg) == 0 {
-			klog.Fatalf("error: --config is required")
+			klog.Error("error: --config is required")
```

Comment:
But don't we need to stop the execution in case of an error?

Reply:
Well the execution is stopped, but it's true that perhaps `klog.Exit` would be better in this case.

Reply:
Thanks